tests/processing/test_create_plate_validator.py

```diff
@@ -347,10 +359,12 @@ def test_validate_adds_single_error_when_multiple_samples_have_the_same_uuid(sub
     subject.validate()
 
     add_error.assert_called_once_with(
-        RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE,
-        f"Sample UUID {sample_uuid} exists more than once in the message.",
-        sample_uuid=sample_uuid,
-        field=FIELD_SAMPLE_UUID,
+        CreatePlateError(
+            origin=RABBITMQ_CREATE_FEEDBACK_ORIGIN_SAMPLE,
+            description=f"Sample UUID {sample_uuid} exists more than once in the message.",
```

Comment:
If we change the description we have to change also the code of the test, seems another point in favour of having error codes :)

Reply:
Agreed